docs/architecture/porting-existing-aspnet-apps/example-migration-eshop.md

```diff
@@ -320,7 +319,7 @@ Then replace the content of the footer in the layout with the following code:
 <section class="col-sm-6">
     <img class="esh-app-footer-text hidden-xs" src="~/images/main_footer_text.png" width="335" height="26" alt="footer text image" />
     <br />
-<small>@eShopPorted.Startup.MachineName - @eShopPorted.Startup.StartTime.ToString() UTC</small>
+<small>@Environment.MachineName - @eShopPorted.Startup.StartTime.ToString() UTC</small>
```

Comment:
Maybe use `Process.GetCurrentProcess().StartTime` to avoid needing to store a static in Startup.cs?

In other scenarios where a method like that isn't available, a more DI-friendly architecture might be to have a stateful service registered as a singleton and request that, as needed, to set or update state. That will allow views to not depend directly on startup.

Reply:
That's a good idea - I'll use `Process.GetCurrentProcess().StartTime`, thanks!